src/components/sections/integration/single-page/bento/bento.styles.ts

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import Image from 'next/image';
```

Comment:
@RBoraurito, Let's use the Image element I recently added. You can check some places for usage references:

1. `src/components/modules/blog-card/blog-card.styles.ts`
`src/components/modules/blog-card/index.tsx#L21`

2. `/src/components/sections/blog/overview-hero/index.tsx#L36`

```suggestion
import { Image } from '@/components/elements/image';
```